tests/generate_status_reference_data.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# Copyright (c) 2018 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import argparse
+import os
+
+import signac
+from define_status_test_project import _TestProject
+from test_project import redirect_stdout
+
+import flow
+import flow.environments
+from flow.scheduling.fake_scheduler import FakeScheduler
+
+PROJECT_NAME = "StatusTest"
+ARCHIVE_DIR = os.path.normpath(
```

Comment:
I think this would make more sense if it were named `ARCHIVE_PATH`. It is a file, not a directory. If you agree, please make the same change to the template tests and update the CLI help messages referring to this variable at the bottom.
```suggestion
ARCHIVE_PATH = os.path.normpath(
```